docs/reference/items/inventory.mdx

```diff
@@ -9,9 +9,9 @@ import Materials from "../../../src/components/materials";
   <div className="col col--12">
     <h3 id="inventory_slots">Inventory Slots</h3>
   </div>
-  <div className="col col--6">
+  <div className="col col--5">
```

Comment:
What did changing this do? I think the text on the right of the picture should be a little closer, if possible.

Reply:
It decreased the gap between the inventory screenshot on the left and the paragraph on the right. Maybe it might be appearing differently based on your screen size? I only tested for 1080p and iPhone 12 1170x2532, which appeared fine

Before:
![image](https://user-images.githubusercontent.com/20259871/229837523-98b8ddfa-73a6-4ab7-9349-02137e8f832d.png)

![image](https://user-images.githubusercontent.com/20259871/229837555-0d05c794-7a6a-4749-b078-25d78e9f6b11.png)